docs/Project.toml

```diff
@@ -2,9 +2,11 @@
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 LaTeXStrings = "b964fa9f-0449-5b57-a5c2-d3ea65f4040f"
+Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
```

Comment:
Could you please add a compat entry below with the required version of Literate.jl? If you are unsure which version is the absolute minimum version, just put the one you are using right now there.